app/Console/TaskOverdueNotificationCommand.php

```diff
@@ -180,7 +180,7 @@ public function sendUserOverdueTaskNotificationsToManagers(array $manager, array
         $this->userNotificationModel->sendUserNotification(
             $manager,
             TaskModel::EVENT_OVERDUE,
-            array('tasks' => $tasks, 'project_name' => $tasks[0]['project_name'])
+            array('tasks' => $tasks, 'project_name' => $tasks[0]['project_name'], 'project_id' => $tasks[0]['project_id'])
```

Comment:
Why do add this property here? it should be also added in other methods if really needed.

Reply:
To have it to calculate the url of Project in the notification